packages/next/src/client/components/react-dev-overlay/internal/components/CodeFrame/CodeFrame.tsx

```diff
@@ -25,11 +25,11 @@ export const CodeFrame: React.FC<CodeFrameProps> = function CodeFrame({
       .reduce((c, n) => (isNaN(c) ? n.length : Math.min(c, n.length)), NaN)
 
     if (prefixLength > 1) {
-      const p = ' '.repeat(prefixLength)
       return lines
         .map((line, a) =>
           ~(a = line.indexOf('|'))
-            ? line.substring(0, a) + line.substring(a).replace(p, '')
+            ? line.substring(0, a) +
+              line.substring(a).replace(`^\\ {${prefixLength}}`, '')
```

Comment:
Can you add a comment explaining what the regex tries to match/replace and why the substring is used, that will help in the future when revisiting the code 🙏